client/app/components/Accordion/index.jsx

```diff
@@ -43,14 +43,19 @@ export const Accordion = ({
     setOpen(!open);
   };
 
+  const openToggle = (event: any) => {
+    if (event.key === 'Enter') {
+      setOpen(!open);
+    }
+  };
   const inputClassNames = () => `${dark ? css.dark : ''} ${large ? css.large : ''}`;
 
   return (
     <div id={`${id}_accordion`} className={inputClassNames()}>
       <div
         className={`accordion ${globalCss.gridRowSpaceBetween} ${css.accordion}`}
         onClick={toggleOpen}
-        onKeyDown={toggleOpen}
+        onKeyDown={openToggle}
```

Comment:
Let's also write a test for this in `client/app/components/Accordion/__tests__/Accordion.spec.jsx`! If you need help on writing this, [here's a similar test for the Modal component](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Modal/__tests__/Modal.spec.jsx#L466).

Reply:
@julianguyen 
I haven't written any test cases yet.
Can you please explain how to write them..

Reply:
Hi @julianguyen 
I have added the test case. Please have a look now, and let me know if anything needs to be changed.